massgov/openmass
# DP-24290 navigation label

**Description:**
Clarify the main navigation labels for screen reader users.

**Jira:** (Skip unless you are MA staff)
DP-24290


**To Test:**
See https://github.com/massgov/mayflower/pull/1743 for the label change and confirm the changes match in openmass.


---

[Peer Review Checklist](https://github.com/massgov/openmass/blob/develop/docs/peer_review_checklist.md)
